docs/manage/fleet/provision/setup.md

```diff
@@ -268,31 +272,33 @@ The following configuration defines the connection information and credentials f
 
 ```json {class="line-numbers linkable-line-numbers"}
 {
-  "manufacturer": "Skywalker",
-  "model": "C-3PO",
-  "fragment_id": "2567c87d-7aef-41bc-b82c-d363f9874663",
-  "hotspot_prefix": "skywalker-setup",
-  "disable_captive_portal_redirect": false,
-  "hotspot_password": "skywalker123",
-  "turn_on_hotspot_if_wifi_has_no_internet": false,
-  "offline_before_starting_hotspot_minutes": "3m30s",
-  "user_idle_minutes": "2m30s",
-  "retry_connection_timeout_minutes": "15m",
-  "turn_on_hotspot_if_wifi_has_no_internet": true,
-  "networks": [
-    {
-      "type": "wifi",
-      "ssid": "otherNetworkOne",
-      "psk": "myFirstPassword",
-      "priority": 30
-    },
-    {
-      "type": "wifi",
-      "ssid": "otherNetworkTwo",
-      "psk": "mySecondPassword",
-      "priority": 10
-    }
-  ]
+  "network_configuration": {
+    "manufacturer": "Skywalker",
+    "model": "C-3PO",
+    "fragment_id": "2567c87d-7aef-41bc-b82c-d363f9874663",
+    "hotspot_prefix": "skywalker-setup",
+    "disable_captive_portal_redirect": false,
+    "hotspot_password": "skywalker123",
+    "turn_on_hotspot_if_wifi_has_no_internet": false,
+    "offline_before_starting_hotspot_minutes": "3m30s",
+    "user_idle_minutes": "2m30s",
+    "retry_connection_timeout_minutes": "15m",
+    "turn_on_hotspot_if_wifi_has_no_internet": true,
+    "networks": [
```

Comment:
There's no "networks" in network_configuration. There's a separate section for "additional_networks" and the format is now an object, not an array. Again, may be easier to just point to the main configuration page, as viam-defaults.json basically takes the exact same layout/format as the cloud config.